docs/README.md

```diff
@@ -1,3 +1,7 @@
+<style>
+.inner {max-width: 800px;}
```

Comment:
Can we add a one-line comment here explaining why? Something like:

`// Increase content width from 640px to 800px for large screens.`

Reply:
Sure. Done.